### PR TITLE
Wutao/setup

### PR DESCRIPTION
**Pull request by wutaomsft**
### Description
Substantially streamlined setup and also updated README

### Checklist:

- [X] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [X] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -376,23 +91,35 @@ Additionally, you must install the [spark-cosmosdb connector](https://docs.datab
 </details>
 
 
-## Setup guide for Docker
 
-A [Dockerfile](tools/docker/Dockerfile) is provided to build images of the repository to simplify setup for different environments. You will need [Docker Engine](https://docs.docker.com/install/) installed on your system.
+## Setup for Experimental 
+<!-- FIXME FIXME 23/04/01 move to experimental. Have not tested -->
```
Comment (miguelgfierro) [inline SETUP.md:96]: I think this is much better to what we had before. One thing that might be interesting to do is one section for developers, just to install for example:
```
pip install .[gpu,examples]
```
or if they want the latest development in staging:
```
git checkout staging
pip install .[gpu,example]
```

**Review by miguelgfierro — APPROVED** [review]
this is great, I added a minor idea

File: SETUP.md (hunk 1 above)
Comment (wutaomsft) [reply SETUP.md:96]: Like the idea! could you add it?  You may have the best idea what to include.

Comment (miguelgfierro) [reply SETUP.md:96]: makes sense, I'll do a new PR

**Pull request merged**